src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -61,6 +67,24 @@
 {% endblock %}
 
 {% block panelContent %}
+
+    <h2>Translation Locales</h2>
+
+    <div class="metrics">
+        <div class="metric">
+            <span class="value">{{ collector.locale }}</span>
+            <span class="label">Locale</span>
+        </div>
+
+        {% if collector.fallbackLocales|length %}
+            <div class="metric">
+                <span class="value">{{ collector.fallbackLocales | join(' | ')  }}</span>
```

Comment:
Here, in addition to showing a `-` when there are no fallback locales, I'd use a comma (`,`) instead of a pipe (`|`) to separate the values.